nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/record/ResultSetRecordSet.java

```diff
@@ -175,10 +175,12 @@ private static DataType getDataType(final int sqlType, final ResultSet rs, final
                     return RecordFieldType.RECORD.getDataType();
                 }
 
-                final String columnName = rs.getMetaData().getColumnName(columnIndex);
```

Comment:
@markap14 In https://github.com/apache/nifi/pull/2386 you added a `readerSchema` to the `ResultSetRecordSet` constructor. I was working with this class and do not need this functionality. I've updated the code so that a null `readerSchema` can be passed, as in my case there is no record reader, just a record writer.

Let me know if you have any concerns. I ran the unit tests for `QueryRecord` and they all ran without failure.